lib/index.js

```diff
@@ -305,7 +305,6 @@ TradeOfferManager.prototype._getFromDisk = function(filenames, callback) {
 				"appid": appID,
 				"contextid": contextID,
 				"start": start,
-				"trading": 1,
```

Comment:
This parameter doesn't seem to have any effect, so it seems to be useless. Steam uses this parameter only when it is loading its own inventory. So it's better to remove it to have the same behavior.